docs/guides/cloud/branch-review.mdx

```diff
@@ -0,0 +1,220 @@
+
+Cypress Branch Review is designed to elevate your pull request (PR) review workflow. It allows you to quickly identify the impact a pull request might have on your test suite in a single view. Compare which tests are failing, flaky, pending, added, or modified between the **<u>source</u>** and **<u>base</u>** branches and prevent the merging of low-quality code.
+
+<DocsImage
+  src="/img/guides/cloud/branch-review/branch-review.jpg"
+  alt="Cypress Branch Review"
+/>
+
+:::info
+
+Cypress Branch Review is currently only available for [GitHub integrated projects](/guides/cloud/integrations/source-control/github). We are working on adding support for GitLab and Bitbucket soon.
+
+:::
+
+## Public Beta Access
+
+:::tip
+
+To access the Branch Review public beta, an admin can simply enable it under "Cypress Labs"<br />in your Organization Settings in Cypress Cloud.
+
+<DocsImage
+  src="/img/guides/cloud/branch-review/labs_beta.png"
+  alt="Enable Branch Review Beta"
+  width="90%"
+/>
+
+:::
+
+## Getting Started
+
+Branch Review works by leveraging the Cypress Cloud GitHub integration to query the GitHub API for **open** pull requests on active branches.
+
+Your project will first need to be connected to a GitHub repository. If you haven't already, follow the steps in [Install the Cypress GitHub app](/guides/cloud/integrations/source-control/github#Install-the-Cypress-GitHub-app) to connect your project to GitHub. Additionally, you may find [passing PR numbers](/guides/continuous-integration/introduction#CI-Build-Information) to runs as environment variables useful when reviewing branches.
+
+To access Branch Review, navigate to the **Branches** tab in the left sidebar and select the branch you want to review. In order to see a comparison, the branch you select must have an open pull request.
+
+{/* TODO: get screenshot with PR tags on the branches */}
+
+<DocsImage
+  src="/img/guides/cloud/branch-review/get-to-branch-review.jpg"
+  alt="Get to Branch Review"
+/>
+
+## Branch Details
+
+### Latest Runs
+
+You can find a callout on the top of the overview tab of the **Latest runs** page that directs to the Branch Review.
+{/* TODO: try different views here */}
+
+<DocsImage
+  src="/img/guides/cloud/branch-review/branch-review-available.png"
+  alt="Branch Review available banner"
+/>
+
+### View Options
+
+"Review" is the default view of the **Branches** tab for projects with linked GitHub repositories via our GitHub integration. "Runs" provides a list filtered by the selected branch for projects without linked repositories or those with an _unknown_ branch.
+
+<DocsImage
+  src="/img/guides/cloud/branch-review/branch-review-views.jpg"
+  alt="Displaying different views for branches"
+/>
+
+### Review Header
+
+The header includes the Git commit message, PR selector, PR status, and base and target branch labels with the Cypress Cloud test run ID number (#).
```

Comment:
We don't use `target` to refer to the source/feature/PR branch anywhere else in these docs from what I can tell. We should update this to whatever broad term we end up using.